### DISCUSSION TITLE
【日本語翻訳】Danbooru.csvの日本語翻訳

### DISCUSSION THREAD

**Discussion by applemango** [DominikDoom/a1111-sd-webui-tagcomplete#265] [Show and tell]
# About
<img width="1651" alt="スクリーンショット 2024-01-09 14 05 22" src="https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/assets/85370582/f0e99fb7-39f6-4ff6-b2a1-d167de477d0f">

**`Danbooru.csv`の日本語訳を作りました。**
[https://github.com/boorutan/booru-japanese-tag](https://github.com/boorutan/booru-japanese-tag)

現在、300単語が人の手によって翻訳され、100Kの単語が機械によって翻訳されています
# How to use
1. [danbooru-machine-jp.csv](https://github.com/boorutan/booru-japanese-tag/blob/main/danbooru-machine-jp.csv)をダウンロードする
2. ダウンロードしたファイルを下のpathを参考にtagsに配置する
```
 ~/stable-diffusiton-webui/extensions/a1111-sd-webui-tagcomplete/tags
```
3. **webui**から**Settings**を開き、サイドバーから**Autocomplete**を選択する
4. 設定項目、**Translation filename**で`danbooru-machine-jp.csv`か`danbooru-jp.csv`を選択する ( 無い場合は横の`🔄`ボタンを押す )
5. **Apply Settings**を押して設定を保存する

これで日本語で`女の子`と入力すると`1girl`などがサジェストに出るようになりました。

人の手によって翻訳された物のみを使いたい場合は[danbooru-jp.csv](https://github.com/boorutan/booru-japanese-tag/blob/main/danbooru-jp.csv)を使用してください

**Comment by DominikDoom** [comment]
ご投稿ありがとうございました！英語と日本語のREADMEにこの投稿へのリンクを追加しました。

**Comment by Howlsfox** [comment]
gooooood job!!
